src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -45,16 +46,23 @@ public RawScriptResult(int? statusCode, object content)
             {
                 foreach (var header in Headers)
                 {
-                    response.Headers.Add(header.Key, header.Value?.ToString() ?? string.Empty);
+                    if (header.Key.Equals("content-type"))
```

Comment:
Please use the `string.Equals` overload that takes a `StringComparison` and specify the comparison type (this should use `StringComparison.OrdinalIgnoreCase`)